plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -30,6 +30,11 @@ vm.is.jre.proceed=\
  Deploying JSPs requires a Java Development Kit (JDK), and so deployment is likely to fail.\n\
  \n\
  Continue with deployment?
+obsolete.runtime.title=Obsolete App Engine Runtime
+obsolete.runtime.proceed=\
+ Project "{0}" is configured to use an obsolete App Engine runtime, and so deployment is likely to fail.\n\
```

Comment:
"is configured to use" --> "specifies" or perhaps just "uses"

"runtime, and so deployment" --> "runtime. Deployment"